data/countries/US.yaml

```diff
@@ -413,7 +448,7 @@ holidays:
       MA:
         name: Massachusetts
         zones:
-          - America/New_York
+          - America/Chicago
```

Comment:
America/New_York was correct here. America/Boston would be even better. What timezone database is this using?

Reply:
it uses moment-timezone. Please see https://github.com/moment/moment-timezone/blob/develop/data/meta/latest.json

Reply:
ok commit is reverted.